preprocessing/tripadvisorPreProc.py

```diff
@@ -0,0 +1,27 @@
+import csv
+f = open('tripadvisor_in-restaurant_sample.csv', encoding="UTF-8")
+output = open('tripadvisor_data_processed_USA.csv', 'w', encoding="UTF-8")
```

Comment:
Let's close these file handles at the end of the script with `f.close()` and `output.close()`

Reply:
Done